### PR TITLE
fix(AIP-191): valid characters in filename

### LINKED ISSUE #947: AIP-191 File names must have valid characters
AIP-191 states [File names must use snake_case](https://google.aip.dev/191) but `api-linter` does not check for invalid characters. 

Valid Character Ranges include:
`a-z`
`A-Z`
`0-9`
`_`

Anything else could be considered invalid. 

As an example, a proto filename should be `test_service.proto` not `test.service.proto`

Question: Should `A-Z` be considered valid?

### PR DESCRIPTION
**Pull request by parthea**
Fixes #947

### CODE REVIEW

File: rules/aip0191/aip0191.go [hunk 1]
```diff
@@ -45,3 +45,4 @@ func hasPackage(f *desc.FileDescriptor) bool {
 }
 
 var versionRegexp = regexp.MustCompile("^v[0-9]+(p[0-9]+)?((alpha|beta)[0-9]*)?$")
+var validCharacterRegexp = regexp.MustCompile("^[a-zA-Z 0-9\\_\\/]*$")
```
Comment (noahdietz) [inline rules/aip0191/aip0191.go:48]:
```suggestion
var validCharacterRegexp = regexp.MustCompile(`^[a-zA-Z 0-9\\_\\/]*$`)
```

I realize that the expression directly above uses `"` but I guess the golang linter wasn't run on that haha

Comment (noahdietz) [inline rules/aip0191/aip0191.go:48]: There is a whitespace in between `A-Z` and `0-9`, is that necessary?

**Review by noahdietz — COMMENTED** [review]
Looks great

File: rules/aip0191/aip0191.go (hunk 1 above)
Comment (parthea) [reply rules/aip0191/aip0191.go:48]: Great catch! I also had a doubt about `A-Z`. Do we allow uppercase?

**Comment by parthea** [comment]
The status checks are stuck. I'm going to try closing and re-opening the PR.

File: rules/aip0191/aip0191.go (hunk 1 above)
Comment (noahdietz) [reply rules/aip0191/aip0191.go]: Probably not, so lets remove it for now, and if it ends up being "OK" we can add it back (always easier to make it less restrictive later).

**Review by noahdietz — APPROVED** [review]
LGTM pending removal of `A-Z` from the "valid regex"

**Comment by noahdietz** [comment]
closing/reopening to trigger Actions

**Pull request merged**